package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@funnode/slack",
-  "version": "1.5.0",
+  "version": "2.0.0",
```

Comment:
let's avoid updating the version until after the PR is approved and is being merged

Reply:
sure, I'll follow that for the other repos

Reply:
Sounds good .. but I'll have to either leave this PR open until the tests update or we'll have to re-add the tests directory in order for the github actions to pass

Reply:
I'm confused now. This PR does not remove the tests directory. It leaves it as it is

Reply:
Hmm, I thought the failure was related to these changes: https://github.com/FunNode/npm-slack/pull/3/checks?check_run_id=337801422. If not, don't worry about it

Reply:
the culprit seems to be
>>> ./cc-reporter after-build --exit-code 0
Error: you must supply a CC_TEST_REPORTER_ID ENV variable or pass it via the -r flag

Reply:
Oh, got it - the action is running on your #master which likely doesn't have the ENV set. It'll work after merging to FunNode master, so we're good - thanks